pkg/registry/generic/rest/proxy.go

```diff
@@ -213,21 +324,20 @@ func (h *UpgradeAwareProxyHandler) defaultProxyTransport(url *url.URL, internalT
 
 // corsRemovingTransport is a wrapper for an internal transport. It removes CORS headers
 // from the internal response.
+// Implements pkg/util/net.RoundTripperWrapper
 type corsRemovingTransport struct {
 	http.RoundTripper
 }
 
-func (p *corsRemovingTransport) RoundTrip(req *http.Request) (*http.Response, error) {
-	resp, err := p.RoundTripper.RoundTrip(req)
+func (rt *corsRemovingTransport) RoundTrip(req *http.Request) (*http.Response, error) {
+	resp, err := rt.RoundTripper.RoundTrip(req)
 	if err != nil {
 		return nil, err
 	}
 	removeCORSHeaders(resp)
 	return resp, nil
 }
 
-var _ = net.RoundTripperWrapper(&corsRemovingTransport{})
```

Comment:
The compiler already enforces this since we use it in a `net.RoundTripperWrapper` context, and the `net` package name now conflicts with the stdlib `net`. If you feel strongly I can alias the package name and add it back.